inkcut/core/svg.py

```diff
@@ -300,7 +354,11 @@ def parse(self, e):
             elif cmd == 'Q':
                 self.quadTo(*params)
             elif cmd == 'A':
-                self.arcTo(*params)
+                x1 = self.currentPosition().x()
+                y1 = self.currentPosition().y()
+                (rx, ry, x_axis_rotation, large_arc_flag, sweep_flag, x2, y2) = params
+
+                self.arc(x1, y1, rx, ry, x_axis_rotation, large_arc_flag, sweep_flag, x2, y2)
```

Comment:
Looking at this again... does `x_axis_rotation` need to be converted to radians here?

https://www.w3.org/TR/SVG/paths.html#PathDataEllipticalArcCommands "and an x-axis-rotation, which indicates how the ellipse as a whole is rotated, in degrees, relative to the current coordinate system".  

All of python's trig functions take radians https://docs.python.org/2/library/math.html#trigonometric-functions.